src/components/project/MembersTab.js

```diff
@@ -176,9 +176,17 @@ class MembersTab extends Component {
       return
     }
     if (owner) {
+      const isRemoveDisabled = _.values(this.props.projectMembers.owners.byId).length <= 1
+
+
       return (
         <IconMenu iconButtonElement={iconButtonElement}>
-          <MenuItem onClick={() => this.removeMember(id, true)}>Remove</MenuItem>
+          <MenuItem
+            disabled={isRemoveDisabled}
```

Comment:
This might be a material ui issue, but the menu item isn't being grayed out. It is just unclickable, which leads to a confusing user experience. Either figure out how to make it obvious that it is intentionally unclickable or add a description that the last user cannot be removed.

Reply:
This actually has to do with the `theme.js` file. In the palette section, you can see `disabledColor: Colors.darkBlack`. This should be changed to some grey color. Maybe `Colors.neutdark_u2`? However, you could also make it even more clear by changing the inner text from `Remove` to something like `{(isRemoveDisabled) ? "Cannot remove last owner" : "Remove"}`